pkg/cloud/aws/actuators/machine/actuator.go

```diff
@@ -289,6 +305,66 @@ func (a *Actuator) Delete(ctx context.Context, cluster *clusterv1.Cluster, machi
 	return nil
 }
 
+func (a *Actuator) cordon(ctx context.Context, cluster *clusterv1.Cluster, machine *clusterv1.Machine, scope *actuators.MachineScope) (error, error) {
+	a.log.Info("Inside cordon()")
+
+	node, err := a.getNodeObject(ctx, cluster, machine, scope)
+	if err != nil {
+		a.log.Info("ERROR: ", err.Error())
+		return nil, err
+	}
+
+	config, err := rest.InClusterConfig()
```

Comment:
This looks like it will only work for self-managed clusters, since remote nodes would not be able to be cordoned using the in-cluster config.

Reply:
Right, I'll switch this bit once I'll get pods evixtion mechanism in place. Thanks for the feedback 👍

Reply:
This has been fixed.